docs: Add “Assistive technology announcements” guidance (#237)
primer/design: content/accessibility/announcements.mdx

---

> title: Announcements

Reviewer: ```suggestion
title: Assistive Technology Announcements
```

Reviewer 2: I updated the suggestion to follow our style guide (sentence case titles)

Author: Thanks @inkblotty and @yaili! I updated the page title in https://github.com/primer/design/pull/237/commits/abef191552d03c4119124f30df944d96d22068ac and the title in the nav in https://github.com/primer/design/pull/237/commits/342dc04cd5aabda26d59296713357107b60bb14c

---

## Overview

> The guidelines below help answer the question “Should I ensure assistive tools output a message in _<this_situation>_?”

Reviewer 2: ```suggestion
These guidelines will help you determine when assistive tools should output a message.
```

Reviewer 2: trying to make it fit with other overviews and remove the "question" format. let me know if I misunderstood the meaning

Author: Your suggested wording is clearer; thanks @yaili! 🙌
